examples/submit-a-form/submit-a-form.js

```diff
@@ -0,0 +1,16 @@
+import { ClientFunction, Selector } from 'testcafe';
+
+fixture `Submit a Form`
+    .page `https://devexpress.github.io/testcafe/example`;
+
+test('Submit a Form', async t => {
+    const submitForm = ClientFunction(() => {
+        document.getElementById('main-form').requestSubmit();
```

Comment:
`document.getElementById('main-form').submit();`

Reply:
`form.submit()` doesn't trigger `onsubmit`, which is required on this page.
So, we have to do either `requestSubmit()` (not available in IE and Safari but we don't run these examples there) or `dispatchEvent(new Event('submit'))`

Reply:
ok